mne_bids/write.py

```diff
@@ -793,6 +793,7 @@ def make_dataset_description(path, name, data_license=None,
                              authors=None, acknowledgements=None,
                              how_to_acknowledge=None, funding=None,
                              references_and_links=None, doi=None,
+                             datasettype='raw',
```

Comment:
Good idea! I would suggest to rename the kwarg. `type` would be the straightforward solution, but of course this leads to a name collision. Can we use `dataset_type`, maybe? `datasettype` is giving me a bit of the shivers :)

```suggestion
                             dataset_type='raw',
```